### PR TITLE
fix broken link in e2e-tests.md

### PR DESCRIPTION
**Pull request by taoyu27**
Signed-off-by: taoyu27 <yutao@chinacloud.com.cn>

### CODE REVIEW

File: contributors/devel/e2e-tests.md [hunk 1]
```diff
@@ -595,7 +595,7 @@ any other labels.
 As each new release of Kubernetes providers new functionality, the subset of
 tests necessary to demonstrate conformance grows with each release. Conformance
 is thus considered versioned, with the same backwards compatibility guarantees
-as laid out in [our versioning policy](../design-proposals/versioning.md#supported-releases).
+as laid out in [our versioning policy](../design-proposals/release/versioning.md#supported-releases).
```
[the remark this answers is not in the document]
Comment (taoyu27) [reply contributors/devel/e2e-tests.md]: ok

[the remark this answers is not in the document]
Comment (taoyu27) [reply contributors/devel/e2e-tests.md]: @cblecker , It is done

[the remark this answers is not in the document]
Comment (cblecker) [reply contributors/devel/e2e-tests.md]: No, I mean replace `../design-proposals/versioning.md#supported-releases` with `/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew` exactly. This removes the ../ relative link. Please also squash your commits :) Thanks!

[the remark this answers is not in the document]
Comment (taoyu27) [reply contributors/devel/e2e-tests.md]: sorry, it is my misunderstanding. fix again. please check it :)

**Comment by cblecker** [comment]
This looks good. Can you please squash your commits? Thank you!
/ok-to-test

**Comment by taoyu27** [comment]
@cblecker , sorry. commit has been squashed :)

**Pull request merged**